src/engraving/dom/hairpin.cpp

```diff
@@ -77,6 +78,8 @@ static const ElementStyle hairpinStyle {
 HairpinSegment::HairpinSegment(Hairpin* sp, System* parent)
     : TextLineBaseSegment(ElementType::HAIRPIN_SEGMENT, sp, parent, ElementFlag::MOVABLE | ElementFlag::ON_STAFF)
 {
+    m_text->setTextStyleType(propertyDefault(Pid::TEXT_STYLE).value<TextStyleType>());
+    m_endText->setTextStyleType(propertyDefault(Pid::TEXT_STYLE).value<TextStyleType>());
```

Comment:
Perhaps can be rewritten more concisely as `m_text->resetProperty(Pid::TEXT_STYLE)`?

Reply:
Using `resetProperty` calls `initTextStyleType` which resets all text style properties. For text lines, these are stored in `TextLineBase`.  Because this is being called in the segment constructor, this happens at layout. This caused issues with opening files, as it would wipe any custom formatting the user had saved on first layout of the score.